src/gui/kb.cpp

```diff
@@ -323,6 +344,7 @@ void Kb::save(){
     settings.setValue("CurrentProfile", currentGuid);
     settings.setValue("Profiles", guids.trimmed());
     settings.setValue("hwLayout", KeyMap::getLayout(_layout));
+    settings.setValue("batteryIndicator", showBatteryIndicator ? "yes" : "no");
```

Comment:
Is there a reason you're not storing a bool? You can simply do 
`settings.setValue("batteryIndicator", showBatteryIndicator);`